src/Workspaces/Core/MSBuild.BuildHost/Rpc/Request.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.CodeAnalysis.Rpc
+{
+    internal sealed class Request
```

Comment:
including MSBuild in the name here would also be ncie IMO.  like MSBuildRPCRequest :)

Reply:
Namespace is now specific enough there's not much ambgiuity anymore.